public/Day 02/script.js

```
@@ -1,107 +1,43 @@
-                        behavior: 'smooth'
-                    });
-                }
-            });
-        });
+// Testimonials Slider
+const slides = document.querySelectorAll(".testimonial");
+let index = 0;
 
-        const scrollToTopBtn = document.getElementById("scrollToTopBtn");
+function showSlide(i) {
+  slides.forEach(slide => slide.classList.remove("active"));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)